frontend/lib/services/meetingService.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:convert';
+import 'package:frontend/models/meeting.dart';
+import 'package:frontend/services/service.dart';
+import 'package:dio/dio.dart';
+
+class MeetingService extends Service {
+
+  Future<List<Meeting>> getMeetings({team : String, company : String, event : int}) async {
+      var queryParameters = {
+        'team' : team,
+        'company' : company,
+        'event' : event
+      };
+
+      Response<String> response = 
+          await dio.get("/meetings", queryParameters : queryParameters);
+
+      if(response.statusCode == 200) {
```

Comment:
Try to change the response handling like in the new version, with try-catch blocks and custom exceptions (my bad, I only put those changes in master branch today). See [member service](https://github.com/sinfo/deck2/blob/master/frontend/lib/services/memberService.dart)

Reply:
Should be solved